Examples update

lexjansen opened the pull request:
- JSON files were update to conform to the schema
-  XPT files were added to the examples
- the examples/sdtm/split folder was deleted
- For illustration purposes several variables that had a DataType='float' in Define-XML had their data type converted to 'decimal' in the JSON examples.

DmitryMK commented:
I have reviewed the files and approved the change. Minor comments:

1) I noticed that sdtm/split subfolder was removed. I can see it in MSG example. Any reason we do not want it to be included?
2) In the VS/LB datasets decimal is used for standardized results. I would actually use decimal for collected results and not for standrdized (converted), because during a conversion we might get non-decimal numbers. Still in the examples I could not find such cases, so for this particular example study it is fine to use decimal for standardized results (although it would make sense to use decimal for collected results too).

lexjansen commented:
Thanks, Dmitri. As far as your comments:

1.  The MSG mentions that the split datasets were used to illustrate splitting because of size issues. That could still happen with JSON, but much less likely. Also my code used the Define-XML look up dataset and variable metadata (OIDs, data types). The split datasets have no metadata in the Define-XML. In wanted to keep it simple.
2. For LB I still kept float for the standardized result. I used decimal for the VS standardized results since I think they were really decimals.  The original results are strings, so I kept them that way. The only data types I converted to decimal were some of the floats. And of course I converted all the date/time formats (and text) to string.  I added a note about the conversion to the examples/README.md.

The programs I used are the same as the code at: https://github.com/lexjansen/sas-papers/tree/master/pharmasug-2022
I plan to create an updated PharmaSUG paper because of the FractionDigits changes.
